modules/serverless-logic-install-kn-workflow-binary-file.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * /serverless/install/serverless-logic-install-kn-workflow-plugin-cli.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-install-kn-workflow-binary-file_{context}"]
+= Installing the OpenShift Serverless Logic Knative Workflow plugin using the binary file
+
+.Prerequisites
+
+* You have installed the Knative (`kn`) CLI.
+* You have installed Podman on your local machine. 
+
+.Procedure
+
+. Download the latest binary file, suitable for your environment, from the link:https://mirror.openshift.com/pub/cgw/serverless-logic/latest/[Serverless Logic download mirror] page.
+
+. Extract the selected Knative Workflow plugin binary file by running the following command:
++
+[source,terminal]
+----
+$ tar xvzf kn-workflow-linux-amd64.tar.gz
+----
+
+. Rename the Knative Workflow plugin binary file to `kn-workflow` by running the following command:
++
+[source,terminal]
+----
+$ mv kn kn-workflow
```

Comment:
this does not correspond to result of `tar xvzf kn-workflow-linux-amd64.tar.gz`. Running this I get: `kn-workflow-linux-amd64` not `kn`